packages/babel-plugin/src/utils/ast-builders.tsx

```diff
@@ -211,7 +211,7 @@ const styledTemplate = (opts: StyledTemplateOpts, meta: Metadata): t.Node => {
     }
   )({
     styleProp,
-    cssNode: t.arrayExpression(opts.sheets.map((sheet) => hoistSheet(sheet, meta))),
+    cssNode: t.arrayExpression(unique(opts.sheets).map((sheet) => hoistSheet(sheet, meta))),
```

Comment:
Q: How we figured out that we have to used `unique`? Any tests was failing? 🤔

Reply:
there was a test failing in class names behaviour https://github.com/atlassian-labs/compiled/blob/14ff96ac20d05d06fb9efea073a4896e358546d2/packages%2Fbabel-plugin%2Fsrc%2Fclass-names%2F__tests__%2Fbehaviour.test.tsx#L89